### DISCUSSION TITLE
Do we have an example of storage class for milvus?

### DISCUSSION THREAD

**Discussion by AliscaCL** [milvus-io/milvus#15989] [Q&A and General discussion]
In tutoral, it only let me to set default storage class. 
But I meet with tons of problem for storage class....
Do we have any example about how to build storage class/ pv/ pvc in distributed cluster?

**Comment by AliscaCL** [comment]
Which kind of pv should I choose?

**Comment by LoveEachDay** [comment]
A storageclass helps you dynamically provision a pv and create a pvc. You can follow the [instructions](https://kubernetes.io/docs/concepts/storage/storage-classes/) and choose a provisioner based on your kubernetes resources and set up a default storageclass.

You can also use `rancher/local-path-provisioner`  to setup a `local-path` storageclass which is the simplest storageclass. Take a look at [here](https://github.com/rancher/local-path-provisioner).
